src/main/java/com/spartronics4915/frc2019/paths/TrajectoryGenerator.java

```diff
@@ -19,8 +19,8 @@
 public class TrajectoryGenerator
 {
 
-    private static final double kMaxVelocity = 130.0;
-    private static final double kMaxAccel = 130.0;
+    private static final double kMaxVelocity = 24.0;
+    private static final double kMaxAccel = 24.0;
```

Comment:
always helps to have a comment indicating units. // 24 ips -> 2 fps
also: i assume we're gonna work on/tune this number?

Reply:
I might play around with increasing these, especially because the "jerky" bug we saw went away. These do not have to be the same, of course.